pkg/server/sandbox_run_test.go

```diff
@@ -82,6 +83,47 @@ func getRunPodSandboxTestData() (*runtime.PodSandboxConfig, *imagespec.ImageConf
 	return config, imageConfig, specCheck
 }
 
+func TestPodAnnotationPassthroughSandboxFilter(t *testing.T) {
```

Comment:
Can we test at `generateSandboxContainerSpec` level?

This test can't make sure that the annotations are actually applied in the OCI spec.

Reply:
And can you change the test to be table driven?

Reply:
done.